src/test/ts/browser/EditorBehaviorTest.ts

```diff
@@ -29,106 +29,114 @@ UnitTest.asynctest('EditorBehaviorTest', (success, failure) => {
 
   const eventStore = EventStore();
 
-  const sTestVersion = (version: Version) => VersionLoader.sWithVersion(
-    version,
-    GeneralSteps.sequence([
-      Logger.t('Assert structure of tinymce and tinymce-react events', Chain.asStep({}, [
-        cRender({
+  VERSIONS.forEach((version) =>
+    context(`TinyMCE (${version})`, () => {
+      it('Assert structure of tinymce and tinymce-react events', async () => {
+        using ctx = await render({
+          cloudChannel: version,
           onEditorChange: eventStore.createHandler('onEditorChange'),
-          onSetContent: eventStore.createHandler('onSetContent')
-        }),
+          onSetContent: eventStore.createHandler('onSetContent'),
+        });
 
         // tinymce native event
         // initial content is empty as editor does not have a value or initialValue
-        eventStore.cEach<SetContentEvent>('onSetContent', (events) => {
+        eventStore.each<SetContentEvent>('onSetContent', (events) => {
           // note that this difference in behavior in 5-6 may be a bug, the team is investigating
           Assertions.assertEq(
             'First arg should be event from Tiny',
             versionRegex.test(version) ? '<p><br data-mce-bogus="1"></p>' : '',
             events[0].editorEvent.content
           );
           Assertions.assertEq('Second arg should be editor', true, isEditor(events[0].editor));
-        }),
-
-        eventStore.cClearState,
-
-        cEditor(cSetContent('<p>Initial Content</p>')),
+        });
+        eventStore.clearState();
 
+        ctx.editor.setContent('<p>Initial Content</p>');
         // tinymce native event
-        eventStore.cEach<SetContentEvent>('onSetContent', (events) => {
+        eventStore.each<SetContentEvent>('onSetContent', (events) => {
           Assertions.assertEq('onSetContent should have been fired once', 1, events.length);
-          Assertions.assertEq('First arg should be event from Tiny', '<p>Initial Content</p>', events[0].editorEvent.content);
+          Assertions.assertEq(
+            'First arg should be event from Tiny',
+            '<p>Initial Content</p>',
+            events[0].editorEvent.content
+          );
           Assertions.assertEq('Second arg should be editor', true, isEditor(events[0].editor));
-        }),
+        });
 
         // tinymce-react unique event
-        eventStore.cEach<string>('onEditorChange', (events) => {
+        eventStore.each<string>('onEditorChange', (events) => {
           Assertions.assertEq('First arg should be new content', '<p>Initial Content</p>', events[0].editorEvent);
           Assertions.assertEq('Second arg should be editor', true, isEditor(events[0].editor));
-        }),
+        });
+        eventStore.clearState();
+      });
 
-        eventStore.cClearState,
-        cRemove
```

Comment:
I assume the logic in Loader.tsx means we don't have to explicitly call `editor.remove` anymore?

Reply:
Only if you create the editor context with the `using` keyword like: `using ctx = await render()`;
You can do `const ctx = await render()`, and it works identically just without automatically calling the dipose callback. So in that case you would need to call `ctx.remove()` yourself.
I've wondered whether enforcing the `using` keyword for disposables can be done with eslint, but haven't looked into it.